ChainReactionBindings/TestBase/Pages/ItemDetailsPage.cs

```diff
@@ -22,6 +22,8 @@ class ItemDetailsPage : BasePage
 		public string EmptyReviewsId = "#crcPDPBottomTabsReviewsLink";

```

Comment:
Please make all the ids private, we should not give access to them outside the PageObject